library/src/main/kotlin/org/contentauth/c2pa/manifest/ManifestBuilder.kt

```diff
@@ -0,0 +1,260 @@
+package org.contentauth.c2pa.manifest
+
+import org.json.JSONArray
+import org.json.JSONObject
+import java.util.*
+
+data class ClaimGenerator(
+    val name: String,
+    val version: String,
+    val icon: String? = null
+)
+
+data class Ingredient(
+    val title: String? = null,
+    val format: String,
+    val instanceId: String = UUID.randomUUID().toString(),
+    val documentId: String? = null,
+    val provenance: String? = null,
+    val hash: String? = null,
+    val relationship: String = "parentOf",
```

Comment:
There can only be 1 parent ingredient in a manifest, thus the Rust SDK defaults to "componentOf." I have the same question if we need to default here or in the Rust SDK.